.NET/Microsoft.Recognizers.Text.Number/Models/AbstractNumberModel.cs

```diff
@@ -9,7 +9,15 @@ namespace Microsoft.Recognizers.Text.Number
     public abstract class AbstractNumberModel : IModel
     {
         // Languages supporting subtypes in the resolution to be added here
-        private static readonly List<string> ExtractorsSupportingSubtype = new List<string> { Constants.ENGLISH, Constants.SWEDISH, Constants.ARABIC };
+        private static readonly List<string> ExtractorsSupportingSubtype = new List<string>
+        {
+                Constants.ENGLISH,
+                Constants.SWEDISH,
+                Constants.ARABIC,
+
+                // Temporarily disabled as existing TestSpec not supporting

```

Comment:
For this type of comment, in the future, please always use the `@TODO` tag in the code comment for ease of search.

Reply:
I'll do while creating new PR for ordinal Today